src/current/_includes/releases/v24.2/v24.2.2.md

```diff
@@ -0,0 +1,19 @@
+## v24.2.2
+
+Release Date: September 23, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-2-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that could result in the inability to garbage collect an [MVCC]({% link v24.2/architecture/storage-layer.md %}#mvcc) range tombstone within a global table. [#130946][#130946]
```

Comment:
maybe add a link to global table: https://www.cockroachlabs.com/docs/v24.2/table-localities#global-tables